### PR TITLE
MyStreamPlayer connector

### PR DESCRIPTION
**Pull request by jbwharris**
This is a new connector I created initially for Bagel Radio. I couldn't find evidence that this streaming service was used by other sites, but wanted to cover my bases in case there were others out there. So it works for https://bagelradio.com and https://mystreamplayer.com URLs currently.

**Additional context**
image

### CODE REVIEW

File: src/core/connectors.js [hunk 1]
```diff
@@ -2317,6 +2317,15 @@ const connectors = [{
 	],
 	js: 'connectors/uwu-radio.js',
 	id: 'uwu-radio',
+}, {
+	label: 'MyStreamPlayer',
```
Comment (inverse) [inline src/core/connectors.js:2321]: Would it be better to make this target specifically bagel radio using the mystreamplayer connector? 

I feel that would be nicer from a user perspective and then have a standalone mystreamone if needed.

cc @hornbuckle as you raised this kinda configuration before.

Comment (jbwharris) [reply src/core/connectors.js:2321]: @inverse That seems reasonable, then it would have Bagel Radio by name in the supported sites too.

Comment (hornbuckle) [reply src/core/connectors.js:2321]: Agreed. Similar to what was done for #3474 and #3477 (and even earlier on #2040).

Comment (jbwharris) [reply src/core/connectors.js:2321]: Just split those off into separate entries

**Review by inverse — APPROVED** [review]
Thanks - much better - removed `allFrames` from MyStreamPlayer since we're running directly on that site.

**Pull request merged**